meetings/2022-09-28.md

```diff
@@ -0,0 +1,187 @@
+* Matthieu: Via Authenticated client now you could say that a client is required (disallow unmitigated access). Consider the following policy:
+
+```ttl
+[]
+  acp:deny acl:Read, acl:Append, acl:Write, acl:Control ;
+  acp:allOf [ acp:client acp:PublicClient ] ;
+  acp:noneOf [ acp:client acp:AuthenticatedClient ] .
+```
+
+* Matthieu: In that example policy, all access is denied except when there is a client. That addresses [your comment](https://github.com/solid/authorization-panel/issues/292#issuecomment-1063600607) in some way because it is a way of denying unmitigated client access.
+* Elf: But isn't the deny a default?
+* Matthieu: The default is that nothing is allowed.
+* ...: Not that everything is denied.
+* ...: Deny acts as a thing that trumps everything that is allowed (something denied cannot be otherwise allowed).
+* Elf: Let's look for example at the [client constraint evaluation](https://github.com/solid/authorization-panel/blob/main/proposals/evaluation/uc-2.5.2-client-constraints.md) (simplified): 
+
+```ttl
+PREFIX acp: <http://www.w3.org/ns/solid/acp#>
+PREFIX acl: <http://www.w3.org/ns/auth/acl#>
+PREFIX ex: <https://example.com/>
+
+ex:ReaderAgentPolicy
+  acp:anyOf ex:BobMatcher ;
+  acp:allow acl:Read .
+
+ex:BobMatcher
+  acp:agent ex:Bob ;
+  acp:client ex:Projectron .
+
+```
+
+* Elf: and discuss the difference between those two matchers:
+
+```ttl
+ex:BobMatcher
+  acp:agent ex:Bob ;
+  acp:client acp:PublicClient .
+```
+
+```ttl
+ex:BobMatcher
+  acp:agent ex:Bob .
+```
+
+* Matthieu: The matcher where Bob and the ~~Public Client~~ Authenticated Client (Note to reader: I thought we were discussing Authenticated agent) are required to satisfy authorization will require Bob using a client.
+* ...: The other matcher will be satisfied if Bob does not use a client as well.
+* Elf: Could you write the second case context
+
+```ttl
+[]
+  acp:target ex:resourceX ;
+  acp:agent ex:Bob .
+```
+
+* Matthieu: The above context being client-less will resolve successfully if acp:client acp:PublicClient is not required.
+* ...: Really it is just a case of unmitigated resource access. If we say unmitigated resource access exists, then the AuthZ domain needs to be able to express this.
+* Matthieu: Sorry I meant "for acp:AuthenticatedClient", I thought we were discussing unmitigated resource access.
+* Elf: In ACP if you ommit acp:agent in the matcher, any agent (incl. no agent) will match? Assuming other properties in the matcher.
+* Matthieu: An empty matcher is never satisfied.
+* ...: This issue [312](https://github.com/solid/authorization-panel/issues/312) is really just about acknowledging the existence of unmitigated client access.
+* ...: I agree that the "Public" named individuals are a bit misleading because they are not matching on a specific attribute, we could consider creating an always matcher instead...
```

Comment:
```suggestion
* ...: I agree that the "Public" named individuals are a bit misleading because they are not matching on a specific attribute. We could consider creating an always matcher instead...
```